quickwit-index-config/src/default_index_config/default_config.rs

```diff
@@ -130,6 +149,9 @@ impl DefaultIndexConfigBuilder {
         if self.store_source {
             builder.add_text_field(SOURCE_FIELD_NAME, STORED);
         }
+        if !self.tag_fields.is_empty() {
+            builder.add_text_field(TAGS_FIELD_NAME, STRING);
```

Comment:
I think we should always add it. One benefit  is that someone could populate it by actually specifying it in the document.
What does @fmassot think?

Reply:
Yeah, it makes sense, I would have agreed if we were not providing a remapping of other fields to the special `_tags`.
But since we are, I think it's better to let the user requests it via remapping.   
I already have the small changes but I will hold till @fmassot  gives us his take.

Reply:
I like the idea of always adding it:
- it means that the user can activate very easily this features by adding new tags in the index config
- as @fulmicoton said it, it could be useful to add tags on one document, basically you can tag your document without even adding a new field in your schema, this can be very flexible
- there is no hidden cost of always adding this field, so there is no risk of always adding this field right now